### DISCUSSION TITLE
请问是否支持连接迁移功能呢？以及4层LB是否有实现方案呢？

### DISCUSSION THREAD

**Discussion by ldmid666** [alibaba/xquic#29] [General]
1. 连接迁移功能，我看有的quic库目前没有支持，比如流行的quic-go，想知道Xquic是否支持了呢？
2. 原有4层LB无法支持连接id的概念，无法支持连接迁移，那么xquic有相应的解决实现吗？

**Comment by Kulsk** [comment]
1. Migration is not supported, but multipath-quic will be supported.

2. The working group is making their efforts on the quic load balance draft, maybe it's a bad idea to implement a private LB method. XQUIC followed the draft, and LB applications can get the cid when connection is created or cid is changed. _conn_create_notify_  callback function and xqc_conn_update_cid_notify_pt callback function could be used to be notified of cid events.
https://datatracker.ietf.org/doc/draft-ietf-quic-load-balancers/

> 1. 连接迁移功能，我看有的quic库目前没有支持，比如流行的quic-go，想知道Xquic是否支持了呢？
> 2. 原有4层LB无法支持连接id的概念，无法支持连接迁移，那么xquic有相应的解决实现吗？

**Reply by faicker** [reply]
Hi, I am interesting at the multipath-quic. What is the schedule, please? @Kulsk

**Reply by ruiqizhou** [reply]
> Hi, I am interesting at the multipath-quic. What is the schedule, please? @Kulsk

https://github.com/alibaba/xquic/milestones
